<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0">
   <item row="5" column="0" colspan="3">
    <widget class="QTableView" name="tableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbYUnit">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y (band value)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="btnReset">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="cbYUnitDataType">
     <item>
      <property name="text">
       <string>Integer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Float</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Data Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Unit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X (wavelength)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="cbXUnitDataType">
     <item>
      <property name="text">
       <string>Integer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Float</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cbXUnit">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionReset">
   <property name="icon">
    <iconset>
     <normaloff>:/images/themes/default/mActionUndo.svg</normaloff>:/images/themes/default/mActionUndo.svg</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="speclibresources.qrc"/>
 </resources>
 <connections/>
</ui>
